test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_torch_inductor_inference.py

```diff
@@ -0,0 +1,288 @@
+    ec2_performance_pytorch_inference(
+        pytorch_inference_graviton,
+        ec2_instance_type,
+        ec2_connection,
+        region,
+        suite,
+        precision,
+    )
+
+
+#@pytest.mark.parametrize("ec2_instance_type", ["p3.2xlarge", "g5.4xlarge", "g4dn.4xlarge"], indirect=True)
+#@pytest.mark.parametrize("suite", ["huggingface", "timm", "torchbench"])
+@pytest.mark.parametrize("ec2_instance_type", ["g4dn.4xlarge"], indirect=True)
+@pytest.mark.parametrize("suite", ["torchbench"])
+@pytest.mark.parametrize("precision", ["float32"])
+def test_performance_ec2_pytorch_inference_gpu(ec2_instance_type, suite, precision, pytorch_inference, ec2_connection, region, gpu_only):
+    _, image_framework_version = get_framework_and_version_from_tag(
+        pytorch_inference)
+    if Version(image_framework_version) in SpecifierSet("<2.0"):
+        pytest.skip("skip the test as torch.compile only supported after 2.0")
+    ec2_performance_pytorch_inference(
+        pytorch_inference,
+        ec2_instance_type,
+        ec2_connection,
+        region,
+        suite,
+        precision,
+    )
+
+
+def ec2_performance_pytorch_inference(image_uri, instance_type, ec2_connection, region, suite, precision):
+    is_gpu = re.search(r"(p3|g4|g5)", instance_type)
+    is_graviton = re.search(r"(c6g|c7g|m7g)", instance_type)
+    device = "cuda" if is_gpu else "cpu"
+    docker_cmd = "nvidia-docker" if is_gpu else "docker"
+    ec2_local_dir = os.path.join("/home/ubuntu", "results")
+    repo_name, image_tag = image_uri.split("/")[-1].split(":")
+
+    LOGGER.info(f"ec2_performance_pytorch_inference params "
+                f"image_uri:{image_uri}"
+                f"instance_type:{instance_type}"
+                f"ec2_connection:{ec2_connection}"
+                f"region:{region}"
+                f"suite{suite}"
+                f"precision:{precision}"
+                )
+
+    # Make sure we are logged into ECR so we can pull the image
+    ec2_connection.run(
+        f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
+
+    ec2_connection.run(f"{docker_cmd} pull -q {image_uri} ")
+    ec2_connection.run(f"mkdir -p /home/ubuntu/results")
+    log_file = f"inductor_benchmarks_{instance_type}_{suite}.log"
+
+    test_cmd = (f"python benchmarks/dynamo/runner.py"
+    f" --suites={suite}"
+    f" --inference"
+    f" --dtypes={precision}"
+    f" --compilers=inductor"
+    f" --output-dir=/root/pytorch/logs_{suite}"
+    f" --extra-args=\"--output-directory=./\""
+    f" --device {device}"
+    f" --no-update-archive"
+    f" --quick"
+    f" --no-gh-comment 2>&1 | tee {log_file}")
+
+    clone_pytorch = (f"git clone --branch v2.0.0 --recursive --single-branch --depth 1 "
+                     f"https://github.com/pytorch/pytorch.git")
+    clone_torchbench = "GIT_CURL_VERBOSE=1 GIT_TRACE=1 git clone --verbose https://github.com/pytorch/benchmark.git"
+    install_prereq = "pip install -U numpy"
```

Comment:
The DLC container already has numpy. Since the torchbench version is frozen, why do we need to update numpy to the latest version ?